<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PresInfo</class>
 <widget class="QMainWindow" name="PresInfo">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>377</width>
    <height>309</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>President Info</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="verticalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>-1</x>
      <y>-1</y>
      <width>381</width>
      <height>261</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_2">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <widget class="QLabel" name="labTermNum">
         <property name="text">
          <string>Enter Term Number:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="leTermNum"/>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QPushButton" name="btGetInfo">
       <property name="text">
        <string>Get Information</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPlainTextEdit" name="pteInfo"/>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>377</width>
     <height>25</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFIle">
    <property name="title">
     <string>FIle</string>
    </property>
    <addaction name="actionQuit"/>
   </widget>
   <addaction name="menuFIle"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionQuit">
   <property name="text">
    <string>Quit</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
